CHANGES.rst

```diff
@@ -10,7 +10,8 @@ Version 3.3.2
 Improvements
 ^^^^^^^^^^^^
 
-- Nothing so far :(
+- Replace deprecated ``pkg_resources`` with using standard library ``importlib``.
+  (:pr:`6273`, thanks :user:`maxnoe`)
```

Comment:
Usually we do not add changelog entries for such things, but if you really prefer to add one for the credits, move it to the "Internal Changes" section

Reply:
I don't really mind, will remove